### DISCUSSION TITLE
redis 캐싱 서버 도입 여부

### DISCUSSION THREAD

**Discussion by jang-199** [jang-199/DoranDoran#209] [Q&A] [answered]
현재 사진 조회 시 파일에서 읽어오는 방식을 사용중.
파일에서 직접 읽어오는 방식 보다 메모리에서 가져와서 사용하는 것이 효율적일 것으로 예상됨.

**Comment by LeeKing2017** [comment, marked as answer]
기본 배경사진처럼 서버에서 저장돼있어 수정할 일이 없는 사진같은 경우 redis 캐싱 서버를 통해서 검색 속도를 높일 수 있을 거라고 생각됩니다.
